### ISSUE TITLE
Unused space at the bottom of the page

### ISSUE THREAD

**Issue by pmontrasio** [robinsloan/perfect-edition#19]
This looks like #13 but it's a different issue.
It's about the space between the text and the menu button at the end of the page: I draw a red graffiti in it in the screenshot below. It could accommodate a few lines of text.
 
![photo_2020-07-29_12-10-38](https://user-images.githubusercontent.com/47483/88787987-4020c880-d184-11ea-93b1-b3de74cbe2d5.jpg)

This is Firefox 68 on Android 10.

**Comment by robinsloan** [comment]
Yeah, that is a big chunk of space! This turns out to be a tricky thing to design consistently across mobile browsers, because of the different amount of browser chrome & the different ways they handle the `vh` unit. On my phone's browsers (iOS, Google Chrome and Safari) that big gap isn't there.

I'd welcome additional research and/or some CSS tinkering to figure out how this can display consistently across many different phone browsers.

Thanks for the note and the screenshot---it's very useful to see.
